vassal-app/pom.xml

```diff
@@ -409,7 +408,6 @@
             <plugin>
               <groupId>org.codehaus.mojo</groupId>
               <artifactId>exec-maven-plugin</artifactId>
-              <version>3.1.0</version>
```

Comment:
Is this really duplicate? Does the other specification actually fix the version _here_?

Reply:
Yes, absolutely.

I the parent POM file, in the PluginsManagement section, line 142, We are already specifying what version of the plugin must be used in all the projects.

That's that way to allow that every module uses the same plugin versions, unless overriden.